config/pod.yaml.hbs

```diff
@@ -30,6 +30,8 @@ spec:
         value: {{container}}
       - name: SD_PIPELINE_ID
         value: "{{pipeline_id}}"
+      - name: SD_ENVIRONMENT
```

Comment:
personally, that VAR name is kind of confusing/misleading to me
not sure if something like `SD_CLUSTER` or `SD_CLUSTER_PREFIX` would make more sense

Reply:
SD_CLUSTER is referred here - https://github.com/screwdriver-cd/screwdriver/blob/master/config/custom-environment-variables.yaml#L359. can change it to SD_PREFIX?

Reply:
updated to SD_BUILD_PREFIX